src/app/components/challenge/challenge.component.html

```diff
@@ -147,7 +147,7 @@
                 </div>
             </div>
         </div>
-        <app-footer [isDash]="true" *ngIf="authService.isAuth"></app-footer>
+        <app-footer [isDash]="true" *ngIf="!authService.isAuth"></app-footer>
     </div>
 </div>
-<app-footer *ngIf="!authService.isAuth"></app-footer>
```

Comment:
@nsjcorps It is happening due to this change, please revert it.

Reply:
Thank you so much @Sanji515 . I used it to bypass the view so I could edit it. It's really  hard doing all these tasks without docker